fdbclient/ClientKnobs.h

```diff
@@ -198,6 +198,8 @@ class ClientKnobs : public KnobsImpl<ClientKnobs> {
 	int HTTP_READ_SIZE;
 	int HTTP_VERBOSE_LEVEL;
 	std::string HTTP_REQUEST_ID_HEADER;
+	bool HTTP_REQUEST_AWS_V4_HEADER;
```

Comment:
SRE needs to toggle it to true to use V4 header? Can you add a comment to clarify it?

Reply:
yes. It is a switch to specify if you need V4 header or the old style header.